EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigi_cfi.py

```diff
@@ -41,3 +41,8 @@
 phase1Pixel.toModify(siPixelDigis, UsePhase1=True)
 phase1Pixel.toModify(siPixelDigisGPU, UsePhase1=True)
 
+# In principle I would like to hide the name 'siPixelDigisGPU', but it
+# is used in test/runRawToDigi_GPU_phase1.py which I also don't want
+# to break
```

Comment:
@makortel , indeed I would prefer that `siPixelDigisGPU` does not appear in the standard reconstruction python file / dump.
What would be a way to make the change only for the `gpu` modifier, without unconditionally creating a `siPixelDigisGPU` module ?

Reply:
Simplest would be prepend underscore to `siPixelDigisGPU` (i.e. make it `_siPixelDigisGPU`). The test configuration quoted in the comment would require some work to be kept functional (but is that needed now that we're proceeding with matrix workflows?).